server-python/meritServer/users/forms.py

```diff
@@ -0,0 +1,13 @@
+from django.contrib.auth.models import User
+
+from .models import MeritUser
+
+class UserForm(forms.ModelForm):
+    class Meta:
+        model = User
+        fields = ('first_name', 'last_name', 'email')
+
+class MeritUserForm(forms.ModelForm):
+    class Meta:
+        model = MeritUser
+        fields = ('student', 'recommender', 'counselor', 'reviewer', 'admin')
```

Comment:
can be changed to a type field => one of many, user can be more than one type

Reply:
This file shouldn't even exist.